src/components/charts/common-chart-options.js

```diff
@@ -16,27 +16,13 @@ class ChartOptions {
 
         // These options can be set globally for all charts
         this.options = {
-            colors: theme === 'primary' ? this.constants.primaryTheme : this.constants.alternateTheme,
             chart: {
                 backgroundColor: 'transparent',
                 style: {
                     fontFamily: '"OpenSans", "Helvetica Neue", arial, sans-serif',
                     color: '#222222',
                 },
             },
-            legend: {
```

Comment:
I've moved *all* the legend configuration to `SpecificChartOptions`, even though some of it is general, in order to avoid the merging of the options getting too complicated - it's easier to merge in all the top level values for each key in the options object.